_stbt/core.py

```diff
@@ -2005,9 +2002,7 @@ def _confirm_match(image, position, template, match_parameters, imglog):
         return True
 
     # Set Region Of Interest to the "best match" location
-    roi = image[
-        position.y:(position.y + template.shape[0]),
-        position.x:(position.x + template.shape[1])]
+    roi = image[region.y:region.bottom, region.x:region.right]
```

Comment:
This could be replaced with `_crop`?.  Probably not worth it though, it would only be slower.